Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -2163,7 +2163,7 @@ - (void)didTapConversationItem:(ConversationViewItem *)viewItem
     }];
 }
 
-- (void)didTapConversationItem:(ConversationViewItem *)viewItem quotedMessage:(OWSQuotedReplyModel *)quotedReply
+- (void)didTapConversationItem:(ConversationViewItem *)viewItem quotedReply:(OWSQuotedReplyModel *)quotedReply
```

Comment:
This was causing crashes when tapping on quoted replies.

Reply:
Death by rename. Thanks for finding this.